verticapy/core/vdataframe/_corr.py

```diff
@@ -804,6 +812,63 @@ def corr(
         -------
         obj
             Plotting Object.
+
+        Examples
+        --------
+        Let's begin by importing `VerticaPy`.
```

Comment:
Maybe remove "Let's" from one or two of the example intro sentences. Four in a row is kind of repetitive.